### DISCUSSION TITLE
Does anyone know how to install VitePress using Termux?

### DISCUSSION THREAD

**Discussion by zhanshenxiaomao** [termux/termux-app#3866] [Q&A] [answered]
Does anyone know how to install VitePress using Termux?

**Comment by KebabLord** [comment, marked as answer]
install using npm?
```
pkg install nodejs-lts 
npm install -g vitepress
```

**Reply by zhanshenxiaomao** [reply]
The problem I'm encountering now is that the submission to GitHub is empty.

**Reply by KebabLord** [reply]
- Create a new repo called `zhanshenxiaomao.github.io`
- Install git on termux, and set up github authenticaton on it, if you dunno how to do it, you can follow this [guide](https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/GitHub-SSH-Key-Setup-Config-Ubuntu-Linux)
- `git clone zhanshenxiaomao.github.io` and cd to it
- follow this guide to deploy your vitepress on github pages: https://vitepress.dev/guide/deploy#github-pages
- then make the commit, push the changes and you will be able to access it on https://zhanshenxiaomao.github.io

**Reply by zhanshenxiaomao** [reply]
Thanks. This is indeed a good approach. I'll try it

**Reply by zhanshenxiaomao** [reply]
Success. Success ran into problems again. It would be great if there was a video reference
